src/components/RedBox.js

```diff
@@ -17,7 +17,7 @@ type StackFrame = {
   functionName?: string,
   source?: string,
   args?: any[],
-  evalOrigin?: StackFrame,
+  evalOrigin?: StackFrame
```

Comment:
It wouldn't; eslint (and the airbnb style guide) doesn't support flow. If you're using Flow, you'd need a separate flow-specific eslint config.